mne/beamformer/_dics.py

```diff
@@ -352,9 +351,9 @@ def csd_shapes(x):
         for i in range(len(frequencies) - 1):
             fstep.append(frequencies[i + 1] - frequencies[i])
         if not np.allclose(fstep, np.mean(fstep), 1e-5):
-            warnings.warn('Uneven frequency spacing in CSD object, '
-                          'frequencies in the resulting stc file will be '
-                          'inaccurate.')
+            warn('Uneven frequency spacing in CSD object, '
+                 'frequencies in the resulting stc file will be '
+                 'inaccurate.')
```

Comment:
can we save lines in general? it seems we have more space in many cases because the caller became shorter